rbx_reflector/plugin.lua

```diff
@@ -1,3 +1,7 @@
+if game.Name ~= "defaults-place.rbxlx" then
+	return
+end
+
```

Comment:
Nice, this will help when Roblox Studio's "Reload plugins on file changed` setting is enabled